sharpd/sharp_vty.c

```diff
@@ -416,37 +416,39 @@ DEFPY (install_seg6_routes,
 	return CMD_SUCCESS;
 }
 
-DEFPY (install_seg6local_routes,
-       install_seg6local_routes_cmd,
-       "sharp install seg6local-routes [vrf NAME$vrf_name]\
+DEFPY(install_seg6local_routes, install_seg6local_routes_cmd,
+      "sharp install seg6local-routes [vrf NAME$vrf_name]\
```

Comment:
Please do not do all this white space changes.  Put it back to the way it was originally and add your additional code in